components/calendar/src/iso.rs

```diff
@@ -368,6 +368,68 @@ impl Iso {
         }
     }
 
+    pub(crate) fn fixed_from_iso(date: IsoDateInner) -> i32 {
```

Comment:
nit: Please document this with something saying that "Fixed dates are represented by counting days since January 1, 1, with January 1, 1 being Fixed Date 1"